go.mod

```diff
@@ -20,21 +20,15 @@ require (
 	github.com/lightninglabs/neutrino v0.12.1
 	github.com/lightningnetwork/lnd/ticker v1.0.0
 	github.com/stretchr/testify v1.5.1
-	golang.org/x/crypto v0.0.0-20200510223506-06a226fb4e37
+	golang.org/x/crypto v0.0.0-20200622213623-75b288015ac9
 	golang.org/x/net v0.0.0-20200520004742-59133d7f0dd7
 	google.golang.org/genproto v0.0.0-20190201180003-4b09977fb922 // indirect
 	google.golang.org/grpc v1.18.0
 )
 
 replace github.com/btcsuite/btcwallet/walletdb => ./walletdb
 
-replace github.com/btcsuite/btcwallet/wtxmgr => ./wtxmgr
-
-replace github.com/btcsuite/btcwallet/wallet/txauthor => ./wallet/txauthor
-
-replace github.com/btcsuite/btcwallet/wallet/txrules => ./wallet/txrules
-
-replace github.com/btcsuite/btcwallet/wallet/txsizes => ./wallet/txsizes
+replace github.com/btcsuite/btcd => github.com/btcsuite/btcd v0.22.0-beta.0.20210803133449-f5a1fb9965e4
```

Comment:
I need to add this as otherwise one of the modules would bump into those bogus tags we recently removed. I should be able to remove this one I update the top-level modules in the next PR.

Reply:
I guess that next PR would also be the btcec/v2 stuff?  Trying to update our app's deps and it looks like the next thing it complains about is

```
    github.com/btcsuite/btcwallet/waddrmgr imports
    github.com/btcsuite/btcd/btcec: module github.com/btcsuite/btcd@latest found (v0.18.1), but does not contain package github.com/btcsuite/btcd/btcec
```

I thought I'd be able to have my app require btcd `master`, but that would seem to be impossible since on master it had a btcec/v2 but no btcec(/v1)